packages/flutter_tools/lib/src/version.dart

```diff
@@ -254,6 +257,8 @@ class FlutterVersion {
     try {
       final String branch = _channel == 'alpha' ? 'alpha' : 'master';
       final DateTime remoteFrameworkCommitDate = DateTime.parse(await FlutterVersion.fetchRemoteFrameworkCommitDate(branch));
+      // fire and forget since nothing subsequent depends on it. Hope there's a flush/sync mechanism.
```

Comment:
Looks like a bug. I think we should `await`. I usually leave a comment if I'm dropping a `Future` on the floor intentionally.